examples/css/create_cluster.py

```diff
@@ -0,0 +1,43 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+'''
+Create CSS cluster
+'''
+import openstack
+
+openstack.enable_logging(True)
+conn = openstack.connect(cloud='otc')
+
+
+attrs = {
+    'name': 'ES-Test',
+    'instanceNum': 4,
+    'instance': {
+        'flavorRef': 'css.large.8',
```

Comment:
It can be a bad idea to use this flavor, see discussion:
https://jira.tsi-dev.otc-service.com/browse/OTCBIGDAI-856

Reply:
It worked for me, but which cluster type would you recommend as an example?

Reply:
I suppose it's okay if it worked :) For the acceptance tests we use `css.medium.8`. But I'm not sure if it has no problems ― we have basically taken the first flavor with which cluster creation hasn't failed.